src/Calculator/App.xaml

```diff
@@ -165,7 +163,6 @@
                     <SolidColorBrush x:Key="EquationBrush12" Color="#FFF7630C"/>
                     <SolidColorBrush x:Key="EquationBrush13" Color="#FF8E562E"/>
                     <SolidColorBrush x:Key="EquationBrush14" Color="#FF000000"/>
-                    <SolidColorBrush x:Key="DividerBrush" Color="#33000000"/>
```

Comment:
You probably still need this for the divider between the equations and the variables

Reply:
fixed!